.github/actions/flaky-tests-report/action.yml

```diff
@@ -55,7 +57,16 @@ runs:
         GITHUB_TOKEN: ${{ inputs.github_token }}
       if: github.event_name == 'pull_request'
 
+    - name: Cleanup uploaded partial reports
```

Comment:
I think i've seen this step couple of times - would be nice to create composite action for it, (non-blocking)